system-test/firestore.js

```diff
@@ -23,7 +23,13 @@ let DocumentReference = require('../src/reference').DocumentReference;
 let DocumentSnapshot =
     require('../src/document')(DocumentReference).DocumentSnapshot;
 
-let version = require('../../package.json').version;
+let version;
+try {
+  version = require('../../package.json').version;
+} catch (e) {
+  version = require('../package.json').version;
+}
```

Comment:
uhh... this seems comment-worthy?

Reply:
This will go away when I merge to master. We just replaced this hack with `pkg-up`.